app/celery/contact_information_tasks.py

```diff
@@ -34,7 +34,7 @@ def lookup_contact_info(self, notification_id):
                 f"The task lookup_contact_info failed for notification {notification_id}. "
                 f"{notification.notification_type} is not supported")
 
-    except VAProfileRetryableException as e:
+    except (Timeout, VAProfileRetryableException) as e:
         if can_retry(self.request.retries, self.max_retries, notification_id):
             current_app.logger.warning("Unable to get contact info for notification id: %s, retrying", notification_id)
             raise AutoRetryException('Found VAProfileRetryableException, autoretrying...', e, e.args)
```

Comment:
```suggestion
            raise AutoRetryException(f'Found {type(e).__name__}, autoretrying...', e, e.args)
```
Since it's now more than just the `VAProfileRetryableException`.